### ISSUE TITLE
vscode only ? 

### ISSUE THREAD

**Issue by steflen** [Quramy/typescript-eslint-language-service#32]
does this add any value to other IDEs like intelliJ webstorm ?  

gg wp ;)

**Comment by AObuchow** [comment]
Yes it adds value to Eclipse IDE through the Wild Web Developer plugin. See [here](https://github.com/eclipse/wildwebdeveloper/blob/master/RELEASE_NOTES.md#eslint-and-typescript-eslint-support).

Big thanks to @Quramy for your work :)

**Comment by wavded** [comment]
Should work wherever tsserver is supported. I have it running on Vim (vim-ale or coc.nvim both work). Thanks @Quramy, seems to work much faster than what I had before (running eslint on file changes).

**Comment by Quramy** [comment]
> seems to work much faster than what I had before (running eslint on file changes)

Yes. My plugin skips lexing and parsing because parsed AST is provided by tsserver.
My plugin only does convert from TypeScript AST to ESLint AST and run ESLint engine using the AST.

https://quramy.github.io/ts-eslint-talk/#slide=18
